src/components/main/ItemsTable.tsx

```diff
@@ -273,7 +272,7 @@ const ItemsTable = ({
       <GraaspTable
         id={tableId}
         columnDefs={columnDefs}
-        tableHeight={ITEMS_TABLE_CONTAINER_HEIGHT}
+        // tableHeight={ITEMS_TABLE_CONTAINER_HEIGHT}
```

Comment:
The issue with the table height is located in graasp/ui i think. Using a responsive table height would be best, but I am not sure whether ag-grid supports it. Need to investigate.